### PR TITLE
fix 1px of white space in spacer

### PR DESCRIPTION
**Pull request by dernasherbrezon**
Hi,

I've finally fixed annoying white space in spacer :)

### CODE REVIEW

**Comment by yucatio** [comment]
Hi dernasherbrezon.
Thank you for making pull request,
I checked your changes.
It works fine.

File: js/jquery.fn.gantt.js [hunk 1]
```diff
@@ -291,7 +291,7 @@
                 var headerRows = scaleGroupSttings[element.scaleGroup].headerRows;
                 var ganttLeftPanel = $('<div class="leftPanel"/>')
                     .append($('<div class="row spacer"/>')
-                    .css("height", tools.getCellSize() * headerRows)
+                    .css("height", (tools.getCellSize() * headerRows + 1 )) // 1 - for border
```
Comment (yucatio) [inline js/jquery.fn.gantt.js]: no need to add parentheses.
Just 
.css("height", tools.getCellSize() * headerRows + 1) // 1 - for border
is fine

**Comment by dernasherbrezon** [comment]
Hi,

I'm not UI developer, so have several questions:

1. Why table was created using div instead of tr/td?
2. During zoom (+/-) div's width  rendered incorrectly. Year's end doesn't match end of 12th month.
screen shot 2016-12-19 at 2 36 09 pm

**Comment by yucatio** [comment]
> Why table was created using div instead of tr/td?

It looks like table but not table.
Each cell is drawn by image. You can find grid.ping in img directory.
By using this image as background image, cells are shown.

> During zoom (+/-) div's width rendered incorrectly. Year's end doesn't match end of 12th 

Can you please make new issue for this? It might be a bug.
I can't reproduce it. Write all information (data and option) in the issue.

**Comment by dernasherbrezon** [comment]
Will open issue soon. Meanwhile, I've created my own gantt diagram by direct html rendering using tables. See https://0space.ru/project looks much better than based on divs. Plus it renders nicely during zooming.

**Comment by yucatio** [comment]
Your page is cool and moves smoothly.
But generally speaking, table tag isn't designed for such kind of usage.
To create charts, using canvas or svg is general solution.
If you need further discussion, create new issue.

**Comment by yucatio** [comment]
I reviewed your code. Please check and fix.

**Comment by dernasherbrezon** [comment]
Good idea! I've created SVG based graphs previously. Will try to render it using SVG.

Will fix issue soon

**Comment by dernasherbrezon** [comment]
Done. Please review

**Comment by yucatio** [comment]
Merged

**Pull request merged**